Libraries/LibIPC/MessageWindows.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2024, Tim Flynn <trflynn89@serenityos.org>
+ * Copyright (c) 2024, stasoid <stasoid@yahoo.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibIPC/Message.h>
+#include <io.h>
+
+namespace IPC {
+
+static void invalid_parameter_handler(wchar_t const*, wchar_t const*, wchar_t const*, unsigned int, uintptr_t)
+{
+}
+// Make _get_osfhandle return -1 instead of crashing on invalid fd in release (debug still __debugbreak's)
```

Comment:
This is intended behavior, I think it is fine for debug. If you want to change it see: https://learn.microsoft.com/en-us/cpp/c-runtime-library/parameter-validation?view=msvc-170